tests/NClient/NClient.Tests/ClientTests/TimeoutClientTest.cs

```diff
@@ -14,11 +15,11 @@ namespace NClient.Tests.ClientTests
     public class TimeoutClientTest
     {
         [Test]
-        public void TimeoutClient_GetWithHttpClientTimeout_ThrowTaskCanceledException()
+        public void TimeoutClient_GetWithHttpClientTimeout_ThrowOperationCanceledException()
         {
             const int id = 1;
             using var api = TimeoutApiMockFactory.MockGetMethod(id);
-            var httpClient = new HttpClient { Timeout = 1.Microseconds() };
```

Comment:
This is one of the valid methods to set a timeout. I think we should leave this test in its original form.

Reply:
Fix